azure-pipelines.yml

```diff
@@ -26,3 +26,28 @@ jobs:
   parameters:
     name: Windows
     vmImage: vs2017-win2016
+
+- job: py37_32bit
+  pool:
+    vmImage: ubuntu-18.04
+
+  steps:
+    - script: |
+        docker pull quay.io/pypa/manylinux2014_i686
```

Comment:
I think we are still building manylinux1 wheels so should probably keep that in sync - @TomAugspurger